Source/Microsoft.Teams.Apps.CompanyCommunicator/ClientApp/src/components/NewMessage/newMessage.tsx

```diff
@@ -597,14 +603,20 @@ export default class NewMessage extends React.Component<INewMessageProps, formSt
     }
 
     private onBtnLinkChanged = (event: any) => {
-        if (!event.target.value.toLowerCase().startsWith("https://")) {
-            this.setState({
-                errorButtonUrlMessage: "URL must start with https://"
-            });
-        } else {
+        if (event.target.value === "") {
```

Comment:
Rather than having a new block, let's put this check in the check with the startsWith

Reply:
if (!(url === "" || event.target...)

Reply:
done